### PR TITLE
documentation -- structure tutorial

### LINKED ISSUE #3019: Restructure Data examples in the documentation (used to be in a "Tutorials" category)
There are currently some examples on how to use "StructureData" and "Pseudopotentials" in a category called "Tutorials", see e.g. [https://aiida.readthedocs.io/projects/aiida-core/en/latest/tutorial/index.html](this page).

The section "tutorials" is not really a tutorials section and should be removed. The content should be 
- double checked and verified
- moved inside the page on [working with data types](https://aiida.readthedocs.io/projects/aiida-core/en/latest/working_with_aiida/index.html#data-types)

### PR DESCRIPTION
**Pull request by d-tomerini**
fix #3019

minor changes to address the AiiDA changes to 1.0
Is it worth changing the "fake terminal" parts with a different formatting (ipython)?

@ltalirz I will address here from  tomorrow all the section referring to issue #3019 , without actually moving the files to a different section (just checking for typos and validity of the code)

pseudopotential and plugin development still to be added

### CODE REVIEW

File: docs/source/examples/structure_tutorial.rst [hunk 1]
```diff
@@ -211,6 +211,7 @@ import phase different kinds will be created::
   >>> asecell = ase.Atoms('Fe2')
```
Comment (ltalirz) [inline docs/source/examples/structure_tutorial.rst]: would you mind renaming `asecell` to `atoms` or `ase_atoms`?
that makes it clearer what it represents.

File: docs/source/examples/structure_tutorial.rst [hunk 2]
```diff
@@ -211,6 +211,7 @@ import phase different kinds will be created::
...
   >>> asecell[0].mass = 55.
   >>> asecell[1].mass = 56.
+  >>> asecell.cell = cell # defines a periodic cell
   >>> s = StructureData(ase=asecell)
   >>> for kind in s.kinds:
   >>>     print kind.name, kind.mass
```
Comment (ltalirz) [inline docs/source/examples/structure_tutorial.rst]: Could you replace those python2-only print statements by python3 ones?

File: docs/source/examples/structure_tutorial.rst (hunk 1 above)
Comment (d-tomerini) [reply docs/source/examples/structure_tutorial.rst]: maybe `ase_structure` maybe? we're just setting the atom to the abject

Comment (ltalirz) [reply docs/source/examples/structure_tutorial.rst]: sure

**Comment by d-tomerini** [comment]
update changes to also partially address together #3019 and #3022

File: docs/source/examples/structure_tutorial.rst [hunk 3]
```diff
@@ -11,13 +11,13 @@ With the :py:class:`~aiida.orm.nodes.data.structure.StructureData` class we did
...
 and we simply provide easy
-ways to convert between the ASE and the AiiDA formats. On the other hand, 
+ways to convert between the ASE and the AiiDA formats. On the other hand,
 we tried to define a "standard" format for structures in AiiDA, that can be
 used across different codes.
 
 
-Tutorial
---------
+StructureData examples of usage
```
Comment (ltalirz) [inline docs/source/examples/structure_tutorial.rst]:
```suggestion
How to use ``StructureData``
```

File: docs/source/examples/structure_tutorial.rst [hunk 4]
```diff
@@ -80,17 +80,17 @@ The following line instead::
 would create a site with 90% probability of being occupied by Calcium, and
 10% of being a vacancy.
 
-Utility methods ``s.is_alloy`` and ``s.has_vacancies`` can be used to
+Utility property ``s.is_alloy`` and ``s.has_vacancies`` can be used to
```
Comment (ltalirz) [inline docs/source/examples/structure_tutorial.rst]:
```suggestion
Utility properties ``s.is_alloy`` and ``s.has_vacancies`` can be used to
```

File: docs/source/examples/structure_tutorial.rst [hunk 5]
```diff
@@ -80,17 +80,17 @@ The following line instead::
...
 verify, respectively, if more than one element if given in the symbols list,
 and if the sum of all weights is smaller than one.
 
-.. note:: if you pass more than one symbol, the method ``s.is_alloy`` will 
-  always return ``True``, even if only one symbol has occupancy 1. and 
+.. note:: if you pass more than one symbol, the property ``s.is_alloy`` will
+  always return ``True``, even if only one symbol has occupancy 1. and
```
Comment (ltalirz) [inline docs/source/examples/structure_tutorial.rst]:
```suggestion
  always be ``True``, even if only one symbol has occupancy 1 and
```

**Comment by ltalirz** [comment]
Once we merge, please let us know which issues should be closed.

**Comment by d-tomerini** [comment]
thank you for your review @ltalirz. This fixes #3019

**Pull request merged**